cmd/mesh/manifest-diff.go

```diff
@@ -47,17 +59,20 @@ func manifestDiffCmd(rootArgs *rootArgs, diffArgs *manifestDiffArgs) *cobra.Comm
 		Long:  "The diff-manifest subcommand is used to compare manifest from two files or directories.",
 		Args:  cobra.ExactArgs(2),
 		Run: func(cmd *cobra.Command, args []string) {
+			if diffArgs.selection != "::" && diffArgs.ignore != "" {
+				logAndFatalf(rootArgs, "Cannot specify both the selected and ignored resources.")
+			}
```

Comment:
I don't think these need to be mutually exclusive as long as the order they are applied is documented. I'd propose select before ignore. 
In that case you could do --selectResources Deployment:*:* --ignoreResources *:*:istio-pilot
That would compare all Deployments except the one called istio-pilot.

Reply:
done.